lib/handlers/compile.ts

```diff
@@ -144,15 +144,28 @@ export class CompileHandler {
 
             if (contentType === 'application/json') {
                 bodyData = JSON.stringify(body);
-            }
-
-            if (contentType === 'application/x-www-form-urlencoded') {
+            } else if (contentType === 'application/x-www-form-urlencoded') {
                 bodyData = body;
+            } else {
+                Sentry.captureException(
+                    new Error(`Unexpected Content-Type received by /compiler/:compiler/compile: ${contentType}`),
```

Comment:
I'm pretty sure whatever is leading to an extraneous `bodyData` must be a Content-Type we aren't expecting

Reply:
Possibly, though in local testing with a variety of ill-formed and well-formed-but-wrong-type I couldn't get it to trigger.